src/Nancy/Json/JavaScriptSerializer.cs

```diff
@@ -29,459 +29,49 @@
 namespace Nancy.Json
 {
     using System;
-    using System.Collections;
     using System.Collections.Generic;
-    using System.ComponentModel;
     using System.IO;
-    using System.Linq;
-    using System.Reflection;
-    using System.Text;
-
-    using Nancy.Helpers;
+    using Nancy.Json.Simple;
 
     public class JavaScriptSerializer
     {
         internal const string SerializedTypeNameKey = "__type";
 
-        List<IEnumerable<JavaScriptConverter>> _converterList;
-        List<IEnumerable<JavaScriptPrimitiveConverter>> _primitiveConverterList;
-
-        int _maxJsonLength;
-        int _recursionLimit;
-        bool _retainCasing;
-        JavaScriptTypeResolver _typeResolver;
-        bool _iso8601DateFormat;
+        bool retainCasing;

```

Comment:
Don't think we need this, we can just read it straight of the configuration object